content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,620 @@ keywords:
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/26f4da06-acc4-4aa2-8540-d56cec931f99.png"
+    width="100%"
+  />
+</div>
+
+- **Document citations:** Precise page and section references for PDF documents with automatic highlighting of relevant content areas.
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/05dfd0b0-b3b5-41a5-b916-0f89a8e50dd1.png"
+    width="100%"
+  />
+</div>
 
-When accessing data (whether from widgets on your dashboard, files you have
-uploaded, or external data sources), OpenBB Copilot will always cite the data
-source it referenced when responding to your query:
+- **Web citations:** Full URL references when web pages are utilized.
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citation_example.png" alt="citations openbb copilot" width="80%" height="80%" />
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/6073a901-0005-4927-8951-021243333c48.png"
+    width="100%"
+  />
 </div>
 
-If the widget that was cited is present on your dashboard with the same arguments used by OpenBB Copilot,
-hovering over the citation will give you the option to scroll to that widget.
+### Create widgets from AI output
+
+The widget creation system enables seamless integration of Copilot-generated content into your workspace environment.
+
+This capability transforms temporary conversational outputs into permanent analytical assets that can be referenced, shared, and incorporated into ongoing workflows.
+
+The system supports multiple artifact types including formatted text summaries, interactive data tables with sorting and filtering capabilities, and fully functional charts with preserved interactivity.
+
+- Text widget:
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/517d6934-6d89-44df-ba5b-995520bcb1e6.png"
+    width="100%"
+  />
+</div>
+
+- Table widget:
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/c987377c-1dc9-4a96-b590-3694a6c764b4.png"
+    width="100%"
+  />
+</div>
+
+- Chart widget:
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/9e7f2d80-1bd1-4238-bffc-099598c995a6.png"
+    width="100%"
+  />
+</div>
+
+
+### User feedback loop
+
+The thumbs up and down buttons in the agent output allow the user to provide feedback in terms of their experience with the copilot. That data can be utilized to create a flywheel that allows developers to understand how the AI agent can be improved.
+
+While the thumbs up goes through without getting any data from the user.
```

Comment:
Suggestion: While the thumbs up does not prompt the user for any additional details.

Reply:
The thumbs down asks for clarification of what went wrong.